features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/blocks/stats/application-throttledcounts/ajax/stats.jag

```diff
@@ -0,0 +1,140 @@
+<%
+include("/jagg/jagg.jag");
+var log = new Log();
+(function () {
+    response.contentType = "application/json; charset=UTF-8";
+    var mod, obj, result,
+            action = request.getParameter("action"), 
+            appName = request.getParameter("appName"),
+            apiName = request.getParameter("apiName"),
+            fromDate = request.getParameter("fromDate"),
+            toDate = request.getParameter("toDate"),
+            site = require("/site/conf/site.json"),
+            msg = require("/site/conf/ui-messages.jag"),
+            allProviders = "__all_providers__";
+
+    if (jagg.getUser() == null) {
+        print({
+            error: true,
+            message: 'AuthenticateError'
+        });
+    } else {
+        var APIUsageStatisticsClient = Packages.org.wso2.carbon.apimgt.usage.client.APIUsageStatisticsClient;
+
+        var provider = jagg.getUser().username;
+        var MultitenantUtils = Packages.org.wso2.carbon.utils.multitenancy.MultitenantUtils;
+        var tenantDomain = MultitenantUtils.getTenantDomain(provider);
+        var usageStatClient;
+
+        var currentLocation = request.getParameter("currentLocation");
+        if (currentLocation != null && currentLocation.indexOf("all-statistics") >= 0) {
+            provider = allProviders;
+        }
+
+        if (provider.indexOf("@") == -1) {
+            provider = provider + "@" + tenantDomain;
+        }
+
```

Comment:
Yes there is a possibility. Thanks for pointing and created a JiRA to track the issue: https://wso2.org/jira/browse/APIMANAGER-4507